.github/actions/download-comparison-artifacts/action.yaml

```diff
@@ -24,7 +24,7 @@ runs:
       shell: bash
       run: |
         pip install pygithub requests
-        python ./scripts/download_single_artifact.py -name ${{ inputs.report-artifact-name }} -token ${{ inputs.github-token }} -outdir current_logs
+        python ./scripts/download_single_artifact.py -name ${{ inputs.report-artifact-name }} -repo patrick-rivos/riscv-gnu-toolchain -token ${{ inputs.github-token }} -outdir current_logs
```

Comment:
you deleted `download_single_artifact.py` so this needs to be updated with `download_artifact.py`